web/src/components/Feedback.tsx

```diff
@@ -67,63 +67,68 @@ const Feedback = ({
+          />
+        ) : (
+          <FormControl error={showErrors && rating === null}>
+            <FeedbackButtons rating={rating} setRating={setRating} />
+            {showErrors && rating === null && <FormHelperText>{t('noteFillFeedback')}</FormHelperText>}
+          </FormControl>
+        )}
+        <OptionalHint>({t('common:optional')})</OptionalHint>
         <TextField
-          id='searchTerm'
-          value={searchTerm}
-          onChange={event => setSearchTerm(event.target.value)}
-          label={t('searchTermDescription')}
-          required
-          fullWidth
-          error={showErrors && !searchTerm}
-          helperText={showErrors && !searchTerm ? t('noteFillFeedback') : undefined}
+          id='comment'
+          value={comment}
+          onChange={event => onCommentChanged(event.target.value)}
+          label={t(commentTitle)}
+          variant='outlined'
+          multiline
+          rows={DEFAULT_ROWS_NUMBER}
+          helperText={t('commentDescription', { appName: buildConfig().appName })}
         />
-      ) : (
-        <FormControl error={showErrors && rating === null}>
-          <FeedbackButtons rating={rating} setRating={setRating} />
-          {showErrors && rating === null && <FormHelperText>{t('noteFillFeedback')}</FormHelperText>}
-        </FormControl>
-      )}
-      <OptionalHint>({t('common:optional')})</OptionalHint>
-      <TextField
-        id='comment'
-        value={comment}
-        onChange={event => onCommentChanged(event.target.value)}
-        label={t(commentTitle)}
-        variant='outlined'
-        multiline
-        rows={DEFAULT_ROWS_NUMBER}
-        helperText={t('commentDescription', { appName: buildConfig().appName })}
-      />
-      <OptionalHint>({t('common:optional')})</OptionalHint>
-      <TextField
-        id='email'
-        value={contactMail}
-        onChange={event => onContactMailChanged(event.target.value)}
-        label={t('contactMailAddress')}
-        variant='outlined'
+        <OptionalHint>({t('common:optional')})</OptionalHint>
+        <TextField
+          id='email'
+          value={contactMail}
+          onChange={event => onContactMailChanged(event.target.value)}
+          label={t('contactMailAddress')}
+          variant='outlined'
+        />
+        <PrivacyFormControl error={showErrors && !privacyPolicyAccepted} required>
+          <PrivacyCheckbox language={language} checked={privacyPolicyAccepted} setChecked={setPrivacyPolicyAccepted} />
+          {showErrors && !privacyPolicyAccepted && <FormHelperText>{t('common:notePrivacyPolicy')}</FormHelperText>}
+        </PrivacyFormControl>
+        <Button onClick={handleSubmit} variant='contained' startIcon={<SendIcon />}>
+          {t('send')}
+        </Button>
+      </Stack>
+      <Snackbar
+        open={snackbarOpen}
+        onClose={handleClose(setSnackbarOpen)}
+        sendingStatus={sendingStatus}
+        autoHideOnSuccess
```

Comment:
added auto hide only for success, because error message requires time to process it